spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniprot/converter/UniProtEntryFeatureConverter.java

```diff
@@ -128,8 +120,10 @@ private void addFeatureCrossReference(
         String dbname = dbType.getName();
 
         List<String> xrefIds = UniProtEntryConverterUtil.getXrefId(xrefId, dbname);
-        document.crossRefs.addAll(xrefIds);
-        document.databases.add(dbname.toLowerCase());
+        if (!dbType.equals(UniprotKBFeatureDatabase.CHEBI)) {
+            document.crossRefs.addAll(xrefIds);
+            document.databases.add(dbname.toLowerCase());
+        }
         document.content.addAll(xrefIds);
```

Comment:
Hi Leo,
Sorry, I didn't understand the change. Isn't it about indexing dbSNP as a cross ref? Earlier we were already adding all the cross ref then why this issue?